### DISCUSSION TITLE
Ability to add an image as Panel Background

### DISCUSSION THREAD

**Discussion by RebornTrack970** [pelican-dev/panel#527] [Ideas]
The ability to add a png image as background behind the panel through the admin settings tab.

(And maybe a feature where users can set a background image behind a server.)
 Example:
![image](https://github.com/user-attachments/assets/b640b12b-77b8-4b22-8c54-904b0d77abe7)

**Comment by rmartinoscar** [comment]
I believe this should be a plugin more then a feature like for Pterodactyl

**Reply by RebornTrack970** [reply]
perhaps, but at least putting background image for the panel should be a default feature in my opinion, it is a simple feaure but would be nice to add a galaxy img or smth thematic.
